charter.html

```diff
@@ -167,9 +167,9 @@ <h2>Scope</h2>
             processing of personal information.  Ways in which new features might
             enable inappropriate processing include (but are not limited to)
             enabling
-            of <a href="https://w3ctag.github.io/privacy-principles/#hl-recognition-cross-site">cross-site
+            of <a href="https://www.w3.org/TR/2022/DNOTE-privacy-principles-20221214/#hl-recognition-cross-site">cross-site
```

Comment:
Definitely good to use a version in TR space, rather than a github.io version.

If you want to refer to a date-specific version, then I think this February 2023 URL would be best. This is a Draft Note publication and the one that the TAG has indicated interest in getting Wide Review on:
https://www.w3.org/TR/2023/DNOTE-privacy-principles-20230223/

Reply:
```suggestion
            of <a href="https://www.w3.org/TR/2023/DNOTE-privacy-principles-20230223/#dfn-cross-site-recognition">cross-site
```